packages/webui/src/components/Authorization.tsx

```diff
@@ -105,7 +105,7 @@ export default function Authorization({ value }: AuthorizationProps) {
         }
       })()}
       {tokenState !== TokenState.Minimal && (
-        <div className={tw('flex flex-row gap-2 bg-slate-100 px-4 pt-4')}>
+        <div className={tw('flex flex-row gap-2 bg-gray-200 px-4 pt-4')}>
```

Comment:
It would be nice if this had a hover state which re-enforced that the data is interactive.  I.e., in the original design, this would have a hove state like this:

![image](https://github.com/FormidableLabs/envy/assets/17857833/44b0d114-f689-4f7c-9015-9b75b8fe3cdb)